fetch.bs

```diff
@@ -6653,6 +6689,153 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+       <var>request</var>'s <a for=request>origin</a>, then increment
+       <var>totalScheduledDeferredBytesForOrigin</var> by <var>length</var>.
+
+       <li><p>If <var>totalScheduledDeferredBytesForOrigin</var> is greater than 64 kilobytes, then
+       throw a "{{QuotaExceededError}}" {{DOMException}}.
+      </ol>
+     </li>
+    </ol>
+   </li>
+  </ol>
+
+ <li><p>Set <var>request</var>'s <a for=request>service-workers mode</a> to "<code>none</code>".
+
+ <li><p>Set <var>request</var>'s <a for=request>keepalive</a> to true.
+
+ <li><p>Let <var>deferredRecord</var> be a new <a>deferred fetch record</a> whose
+ <a for="deferred fetch record">request</a> is <var>request</var>.
+
+ <li><p><a for=list>Append</a> <var>deferredRecord</var> to <var>request</var>'s
+ <a for=request>client</a>'s <a for=fetch>fetch group</a>'s
+ <a for="fetch group">deferred fetch records</a>.
+
+ <li>
+  <p>If <var>activateAfter</var> is not null, then run the following steps <a>in parallel</a>:</p>
+
+  <ol>
+   <li><p>The user agent should wait until <var>activateAfter</var> milliseconds have passed,
+   or until the user agent has a reason to believe that it is about to lose the opportunity to
+   execute scripts, e.g. when the browser is moved to the background, or when <var>request</var>'s
+   <a for=request>client</a> is a <code>Document</code> that had a "<code>hidden</code>"
+   <a for=Document>visibility state</a> for a long period of time.
+
+   <li><p>The user agent may wait for a further <a>implementation-defined</a> <a>duration</a>, e.g.
+   in order to batch several requests together or to wait until <a for=request>keepalive</a>
+   requests are complete.
+
+   <li><p><a>Process a deferred fetch</a> given <var>deferredRecord</var>.
+  </ol>
+ </li>
+
+ <li><p>Return <var>deferredRecord</var>.
+</ol>
+</div>
+
+<div algorithm="process-deferred-fetches">
+<p>To <dfn export>process deferred fetches</dfn> given a <a>fetch group</a> <var>fetchGroup</var>:
+
+<ol>
+ <li><p>Let <var>deferredFetchRecords</var> be <var>fetchGroup</var>'s
+ <a for="fetch group">deferred fetch records</a>.
+
+ <li><p>Let <var>fetchGroup</var>'s
+ <a for="fetch group">deferred fetch records</a> be « ».
+
+ <li><p><a for=list>For each</a> <a for="fetch group">deferred fetch record</a>
+ <var>deferredRecord</var> of <var>deferredFetchRecords</var>, <a>process a deferred fetch</a> given
+ <var>deferredRecord</var>.
+</ol>
+</div>
+
+<div algorithm="process-a-deferred-fetch">
+<p>To <dfn>process a deferred fetch</dfn> <var>deferredRecord</var>:
+<ol>
+ <li><p>If <var>deferredRecord</var>'s <a for="deferred fetch record">invoke state</a> is not
+ "<code>deferred</code>", then return.
+
+ <li><p>Set <var>deferredRecord</var>'s <a for="deferred fetch record">invoke state</a> to
+ "<code>activated</code>".
+
+ <li><p><a for=/>Fetch</a> <var>deferredRecord</var>'s
+ <a for="deferred fetch record">request</a>.
+</ol>
+</div>
```

Comment:
Eating a newline before the `<h2>` here.

Reply:
Fixed